### DISCUSSION TITLE
Runs are not grouping with Parallelization

### DISCUSSION THREAD

**Discussion by FlorianWoelki** [sorry-cypress/sorry-cypress#640] [General]
Hi there,

I was just asking if this is only an issue on my end, but sometimes the runs are not grouping with parallelization enabled. This issue appears only sometimes (mostly after a couple of days when nothing happened to the director). The following screenshot demonstrates this behavior:

<img width="1115" alt="image" src="https://user-images.githubusercontent.com/20746070/188412552-6220ce66-312e-459c-bd84-665b26d31638.png">

The `build-ci` id is the correct one. I am using BitBucket pipelines and the number of visible runs for the project corresponds to the number of parallel running E2E-tests. Could it be an issue in Mongo?

Thanks!

**Comment by agoldis** [comment]
Most chances MonogDB index disappeared after an upgrade / DB migration. Please make sure the indexes are as in https://github.com/sorry-cypress/sorry-cypress/blob/master/packages/mongo/src/db.ts#L72

**Reply by FlorianWoelki** [reply]
Thanks that solved it. :)
